README.md

```diff
@@ -12,26 +12,27 @@ A Debian-based distribution.
 
 ## Role Variables
 
-| Name                      | Mandatory / Default | Description                                                                              |
-|:-------------------------:|:-------------------:|:----------------------------------------------------------------------------------------:|
-| `admins`                  | `[]`                | list of admin users, see [User configuration](#User configuration)                       |
-| `locales`                 | `[en_US.UTF-8]`     | list of locales to install                                                               |
-| `keyboard_layout`         | `us,de`             | TTY keyboard layout                                                                      |
-| `default_language`        | `en_US.UTF-8`       | default language                                                                         |
-| `apt_repository_versions` | `[main, universe]`  | List of apt repository versions to enable                                                |
-| `common_debian_sources`   |                     | Verbatim sources.list definition for Debian. Overrides the usual sources.list generation |
-| `common_ubuntu_sources`   |                     | Verbatim sources.list definition for Ubuntu. Overrides the usual sources.list generation |
+| Name                      |        Mandatory / Default        | Description                                                                              |
+|:--------------------------|:---------------------------------:|:-----------------------------------------------------------------------------------------|
+| `admins`                  |                `[]`               | list of admin users, see [User configuration](#User configuration)                       |
+| `locales`                 |          `[en_US.UTF-8]`          | list of locales to install                                                               |
+| `keyboard_layout`         |              `us,de`              | TTY keyboard layout                                                                      |
+| `default_language`        |           `en_US.UTF-8`           | default language                                                                         |
+| `apt_repository_versions` |         `[main, universe]`        | List of apt repository versions to enable                                                |
+| `common_debian_sources`   |                                   | Verbatim sources.list definition for Debian. Overrides the usual sources.list generation |
+| `common_ubuntu_sources`   |                                   | Verbatim sources.list definition for Ubuntu. Overrides the usual sources.list generation |
+| `common_packages`         | see [defaults](defaults/main.yml) | Common packages to install                                                               |
```

Comment:
I would recommend introducing an additional variable for custom packages, otherwise we will have to copy the whole list of default packages from here to our group variables.
role and group/host variables are not merged.